hera_mc/autocorrelations.py

```diff
@@ -48,7 +48,7 @@ class HeraAuto(MCDeclarativeBase):
     antenna_number = Column(Integer, primary_key=True)
     antenna_feed_pol = Column(String, primary_key=True)
     measurement_type = Column(String, nullable=False)
-    value = Column(Float, nullable=False)
+    value = Column(Float, nullable=True)
```

Comment:
We should only do this if redis actually gets updated with null correlations. If that doesn't happen, there's no point in doing this.